# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/loading/preloader-css-import.tentative.html
# Commit: 3dfd3c94a105
# Full Hash: 3dfd3c94a105e095aada0b356f1106370de222d3
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-07 15:51:17
# Regressor Bug: 1701828
# File Overlap Count: 1
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/testing/web-platform/tests/loading/preloader-css-import.tentative.html b/testing/web-platform/tests/loading/preloader-css-import.tentative.html
--- a/testing/web-platform/tests/loading/preloader-css-import.tentative.html
+++ b/testing/web-platform/tests/loading/preloader-css-import.tentative.html
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
+<meta charset="utf-8">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script>
     var t = async_test('Imported inline CSS is not blocked on pending CSS');
 </script>
 <link rel=stylesheet href="resources/dummy.css?first-preloader-css-import&pipe=trickle(d1)">
 <script>
     var this_script_is_necessary_to_block_the_inline_style_processing = true;